__performance_tests__/add-data.js

```diff
@@ -1,103 +1,109 @@
 "use strict"
 
 import {measure} from "./measure"
-import produce, {setAutoFreeze, setUseProxies} from "../dist/immer.umd.js"
+import produce, {
+	setAutoFreeze,
+	setUseProxies,
+	enableAllPlugins
+} from "../dist/immer.cjs.production.min.js"
```

Comment:
Do you see some benefit to refer minified prod build in tests? For once it might be harder to run debugger with it. You could simply refer to `../dist` here which will pickup dev build so it would be less verbose.

Reply:
no, these are the performance test, not the unit tests. So they won't really assert anything and arent to be relied upon for correctness. Using a prod build to better reflect raw production performance